tool/resources/org/antlr/v4/tool/templates/codegen/Go/Go.stg

```diff
@@ -1454,6 +1452,10 @@ const (
 // <lexer.name><first(rest(lexer.modes))> is the <lexer.name> mode.
 const <lexer.name><first(rest(lexer.modes))> = 1
 <endif>
+<if(namedActions.members)>
+
+<namedActions.members>
+<endif>
```

Comment:
This partially fixes #53. The PositionAdjustingLexer members were not showing up in this type.